client/src/Components/Dialog.js

```diff
@@ -14,6 +14,10 @@ export default function NavDialog({ open, close }) {
 	const theme = useTheme();
 	const fullScreen = useMediaQuery(theme.breakpoints.down('sm'));
 
+	const closeDialog = () => {
+		return close(false)
+	}
+
```

Comment:
I don't think you need this though, cause `closeDialog `is already defined as `setOpen(false)`

Reply:
yaa, right. its ok now